docs/modules/ROOT/pages/fire_insurance_registration.adoc

```diff
@@ -0,0 +1,23 @@
+= Fire insurance registration  
+
+== Overview
+
+In order to work with _GIF instance_ each component of _fire insurance_ undergoes registration. 
+Registration procedure consists of component proposal tx (by component _owner_) followed by component approval tx (by _GIF instance operator_) with consequent configurations.
+
+Each component has _owner_ (implements `getOwner()` method) and _type_ (implements `getType()` method).
+Prior to registration, _GIF instance operator_ grants component _owner_ a role correspondent to component _type_. 
+
+Components registration order is important: starting with `FireOracle`, followed by `FireRiskpool` and finishing with `FireProduct`. 
```

Comment:
Actually the order is only important because the FireProduct requires the id of the riskpool and oracle for deployment (in the constructor), but the ordering of the oracle and riskpool are independent.